packages/contracts-bedrock/test/scripts/VerifyOPCM.t.sol

```diff
@@ -223,4 +241,38 @@ contract VerifyOPCM_Run_Test is VerifyOPCM_TestInit {
         vm.expectRevert(VerifyOPCM.VerifyOPCM_Failed.selector);
         harness.run(address(opcm), true);
     }
+
+    /// @notice Tests that the script verifies all four contracts have the same contractsContainer address.
+    function test_verifyContractsContainerConsistency_succeeds() public {
+        // Coverage changes bytecode and causes failures, skip.
+        skipIfCoverage();
+
+        // This should succeed with the current setup where all contracts have the same containerAddress.
+        harness.verifyContractsContainerConsistency(opcm);
+    }
+
+    /// @notice Tests that the script reverts when contracts have different contractsContainer addresses.
+    function test_verifyContractsContainerConsistency_mismatch_reverts() public {
```

Comment:
I think this test can be simplified to just:
1. get the `opcmUpgrader`
2. `vm.store(opcmUpgrader, bytes32(uint256(0)), bytes32(uint256(uint160(differentContainer))));

Then you don't need Mock contract or the `vm.etch` call.